internal/cmd/config.go

```diff
@@ -89,19 +90,17 @@ type Config struct {
 	// Error handling configuration
 	Errors errorhandler.Config
 
-	Helm struct {
-		Tiller struct {
-			Version string
-		}
-
-		Home string
-
-		Repositories map[string]string
+	Github struct {
+		Token string
+	}
```

Comment:
This was removed recently. Is this a bad merge conflict resolution?

Reply:
yes